README.md

```diff
@@ -280,9 +264,7 @@ for (int i = 0; i < 10; i++) {
 
 ## Control Structures
 
-Don't use multiple return statements.
```

Comment:
I removed this statement because just a few paragraphs down it explicitly says that you *should* use multiple return statements.